File: C#/백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = new int[4];
            for (int i =0; i<4; i++){
                aa[i] = int.Parse(Console.ReadLine());
            }
            int b = aa[0]*aa[3];
            if((b==64 || b==72 || b==81) && (aa[1]==aa[2])){
                Console.WriteLine("ignore");
            }
            else{
                Console.WriteLine("answer");
            }
        }
    }
}

File: Java/백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int b=in.nextInt(); int c =in.nextInt(); int d=in.nextInt();
        if((a*d==64 || a*d==72 || a*d==81) && b==c){
            System.out.println("ignore");
        }
        else{
            System.out.println("answer");
        }
    }
}